src/localization/en/learn.json

```diff
@@ -15,7 +15,7 @@
   "steps.basicMatchers.description": "The character or word we want to find is written directly. It is similar to a normal search process. For example, to find the word `curious` in the text, type the same.",
 
   "steps.dotCharacter.title": "Dot `.`: Any Character",
-  "steps.dotCharacter.description": "The period `.` allows selecting any character, including special characters and spaces.",
+  "steps.dotCharacter.description": "The period `.` allows selecting any character, including special characters and spaces. Type a period (`.`) in the Regex field to proceed.",
```

Comment:
Good idea, but this suggestion shouldn't include parentheses. In this project, we never use parentheses for what we want the user to type.

Reply:
ah so the parenthetical should be rephrased is what you're suggesting?

Reply:
Actually it should be like this. We can't wrap what we want it to write in parentheses.

```
"The period `.` allows selecting any character, including special characters and spaces. Type a period `.` in the Regex field to proceed."
````